### ISSUE TITLE
load or run 

### ISSUE THREAD

**Issue by wuxindao** [gudzpoz/luajava#209]
使用party.iroiro.luajava.Lua.load 加载一个lua文件的时候，通过lua.get无法获取一个已加载的函数
使用party.iroiro.luajava.Lua.run 可以获取一个已加载的lua文件函数 
请问，使用load的方式，如果最后来统一管理和获取一个函数

**Comment by wuxindao** [comment]
lua.run(xxxx)
lua.get(xx) 这样可以获取到我想要的函数

lua.load(xxxx)
lua.get(xx) nil 无法获取

**Comment by gudzpoz** [comment]
Yes, Lua.load eventually uses lua_load, and it only loads a chunk; it does not run it ([lua.org/manual/5.1/manual.html#lua_load](https://www.lua.org/manual/5.1/manual.html#lua_load)). I guess I just assume people are familiar with Lua C API and I will probably need to add more documentation to the Lua.load methods.

If your Lua files depend on one another, you should [modulize your Lua code](https://www.lua.org/pil/15.html) and use [`require`](https://www.lua.org/manual/5.1/manual.html#pdf-require).
